src/ddmd/expression.d

```diff
@@ -8772,7 +8773,7 @@ extern (C++) final class DotIdExp : UnaExp
                     if (v.type.ty == Terror)
                         return new ErrorExp();
 
-                    if ((v.storage_class & STCmanifest) && v._init)
+                    if ((v.storage_class & STCmanifest) && v._init && !wantsym)
```

Comment:
For parameter names, I prefer the shorter names. Having lines of code longer than 80 columns is a readability problem.